api/v1beta1/module_types.go

```diff
@@ -89,6 +105,10 @@ type KernelMapping struct {
 	// Build enables in-cluster builds for this mapping and allows overriding the Module's build settings.
 	Build *Build `json:"build"`
 
+	// +optional
+	// Sign enables in-cluster signing for this mapping
+	Sign *Sign `json:"sign"`
```

Comment:
as said in the previous PR, i think that the SIGN section should be outside of Build

Reply:
I'm not sure I understand what you mean. In this PR it's alongside build, not in it, so we have `kernelmapping.build` and `kernelmapping.sign` not `kernelmapping.build.sign`.  That was how I interpreted your previosu comments but do you mean it should be outside of `kernelmapping` entirely and we only have one signing section for all kernel versions?

Reply:
I think this is much better. Shall we add `,omitempty` to the `json` tag?